docs/docusaurus/docs/cloud/quickstarts/airflow_quickstart.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_label: 'Quickstart for GX Cloud and Airflow'
+title: 'Quickstart for GX Cloud and Airflow'
+id: airflow_quickstart
+description: Connect GX Cloud to an Airflow Orchestrator.
+---
+
+In this quickstart, you'll learn how to use GX Cloud with Apache Airflow. Apache Airflow is an orchestration tool that allows you to schedule and monitor your data pipelines. For more information about Apache Airflow, see the [Apache Airflow documentation](https://airflow.apache.org/docs/apache-airflow/stable/index.html).
+
```

Comment:
Even being familiar with GX Cloud and Airflow, I still found myself wondering "what are we actually accomplishing with this code?" I'd suggest having a short sentence or two upfront that explicitly spells out for the user what the quickstart does, something along the lines of: "This quickstart implements a simple Airflow DAG that runs an existing GX Cloud Checkpoint on a schedule."

Reply:
@rachhouse great suggestion.